### ISSUE TITLE
fyne package will not build Android applications when sources are not in same directory

### ISSUE THREAD

**Issue by xxxserxxx** [fyne-io/fyne#653]
### Describe the bug:

If sources are in a subdirectory, `fyne package` will not cross-compile and instead reports an error.

### To Reproduce:

Steps to reproduce the behaviour:

```
$ hg clone https://hg.sr.ht/\~ser/healthdiary
destination directory: healthdiary
requesting all changes
adding changesets
adding manifests
adding file changes
added 1 changesets with 4 changes to 4 files
new changesets 65d468f58cb5
updating to branch default
4 files updated, 0 files merged, 0 files removed, 0 files unresolved
$ cd healthdiary
$ fyne package -os android -appID ser1.net.healthdiary -sourceDir ./cmd/diary -icon assets/hdf.png -name "Health Diary"
no buildable Go source files in /home/ser/tmp/healthdiary
$ echo $?
1
```

This works with `-os linux`:

```
$ fyne package -os linux -sourceDir ./cmd/diary -icon assets/hdf.png
$ echo $?
0
```

### Device (please complete the following information):
 - OS: Linux
 - Version: 5.4.15
 - Go version: go version go1.13.7 linux/amd64
 - Fyne version: Fyne doesn't have an (obvious) version function, but git reports it was built from eba8bb2e8629ad0cbe2d4b07a051043a9759e4f3

**Comment by andydotxyz** [comment]
it looks like it is ignoring the -sourceDir parameter.
Can you please confirm that it works if you change into the "cmd/diary" directory?

**Comment by andydotxyz** [comment]
I have not found an easy fix for this issue, this whole area needs work.
The latest version on develop will abort if -sourceDir is set for a mobile build and inform the user to change into the main package dir.

I hope that helps people avoid problems - leaving the ticket open until we properly resolve the issue.

**Comment by xxxserxxx** [comment]
Yes; if I also address #655 and remove version information from the import path, it works if I `cd` to the `cmd/diary` directory and execute the build there.  It also works to copy all of the files into the top-level directory.

I'll file a separate ticket for the version issue, because I can produce a minimal test case.
